command_line/export_best.py

```diff
@@ -0,0 +1,173 @@
+from __future__ import absolute_import, division, print_function
+
+import logging
+import sys
+
+from libtbx.phil import parse
+from libtbx import Auto
+
+logger = logging.getLogger("dials.command_line.export_best")
+
+help_message = """
+This program is used to export the results of dials processing for the strategy
+program BEST.
+
+"""
+
+phil_scope = parse(
+    """
+
+  intensity = *auto profile sum scale
+    .type = choice(multi=True)
+    .help = "Choice of which intensities to export. Allowed combinations:
+            scale, profile, sum, profile+sum, sum+profile+scale. Auto will
+            default to scale or profile+sum depending on if the data are scaled."
+
+  debug = False
```

Comment:
I don't think this debug parameter is used anywhere.

Reply:
Could be passed through `params.debug` to other code? Will look.

Reply:
I'm pretty sure it isn't - and if you remove the parameter and the tests still pass, then it probably isn't used :)

Reply:
Removed